### DISCUSSION TITLE
Mount files to the container or specify what image to use.

### DISCUSSION THREAD

**Discussion by cbvbs** [dflook/terraform-github-actions#343] [Q&A]
Is there anyway we can specify in the action what container to pull? It would be great to build on https://hub.docker.com/r/danielflook/terraform-github-actions-base and add scenario specific setups.

I would like to continue to use these actions but with the introduction of an internal firewall in my company, the default dflook container won't be able to make requests to our GitHub Enterprise server without a private CA loaded.

**Comment by dflook** [comment]
There a few different ways you could customise the container, but unfortunately you can't specify a custom image when running the action. Take a look here for some other options: https://github.com/dflook/terraform-github-actions/blob/main/docs/custom_tools.md

If you just need to add a certificate, TERRAFORM_PRE_RUN might be the easiest way.
